src/controller/postController.js

```diff
@@ -29,7 +29,7 @@ export const createPost = async (req, res) => {
       if(req.file) savePostImage = await uploadToCloud(req.file, res);
 
     const post = await Posts.create({
-      postImage: savePostImage?.secure_url || "https://res.cloudinary.com/da12yf0am/image/upload/v1696850499/pbxwlozt1po8vtbwyabc.jpg",
+      postImage: savePostImage?.secure_url || "https://res.cloudinary.com/da12yf0am/image/upload/v1698131855/z3epyxo5ytlabbvegjiy.jpg",
```

Comment:
Can you allow a post without an image? Like when no image provided or if Cloudinary returns an error you can a post be crested

Reply:
Corrected camelCase